tests/google_sheets_2/test_google_sheets_2.py

```diff
@@ -279,3 +280,24 @@ def test_schema_fields_order(con, ds):
     assert schema_props_keys[0] == 'domain'
     assert schema_props_keys[1] == 'spreadsheet_id'
     assert schema_props_keys[2] == 'sheet'
+
+
+def test_parse_datetime(mocker, con):
+    ds = GoogleSheets2DataSource(
+        name='test_name',
+        domain='test_domain',
+        sheet='Constants',
+        spreadsheet_id='1SMnhnmBm-Tup3SfhS03McCf6S4pS2xqjI6CAXSSBpHU',
+        parse_dates=['a_date'],
```

Comment:
If we expect to have an array here, we should also modify Tucana's form schema (same as you did for the CSV extractor)